[pentair] Some improvements to the pentair documentation. (#7535)
openhab/openhab-addons: bundles/org.openhab.binding.pentair/README.md

Number Salinity                "Salinity [%d ppm]"                            (gPool) { channel = "pentair:intellichlor:1:ic40:salinity" }

Switch Pump_Run                "Pump running [%d]"                            (gPool) { channel = "pentair:intelliflo:1:pump1:run" }
Number Pump_DriveState         "Pump drivestate [%d]"                         (gPool) { channel = "pentair:intelliflo:1:pump1:drivestate" }
Number Pump_Mode               "Pump Mode [%d]"                               (gPool) { channel = "pentair:intelliflo:1:pump1:mode" }
Number Pump_RPM                "Pump RPM [%d]"                                (gPool) { channel = "pentair:intelliflo:1:pump1:rpm" }
Number Pump_Power              "Pump Power [%d W]"                            (gPool) { channel = "pentair:intelliflo:1:pump1:power" }
Number Pump_Error              "Pump Error [%d]"                              (gPool) { channel = "pentair:intelliflo:1:pump1:error" }
Number Pump_PPC                "Pump PPC [%d]"                                (gPool) { channel = "pentair:intelliflo:1:pump1:ppc" }
```

> Here is an example of a complete sitemap, saved as pentair.sitemap.  Adjust the temperature values for metric is so desired.

Reviewer: Can you put each sentence into a separate line and maybe use backticks to surround the file name here and the others above?

```suggestion
Here is an example of a complete sitemap, saved as `pentair.sitemap`.
Adjust the temperature values for metric is so desired.
```

Reviewer 2: Also just to add on to this.
```suggestion
Here is an example of a complete sitemap, saved as `pentair.sitemap`.
Adjust the temperature values to metric if so desired.
```
